src/RadioGroup/Radio.tsx

```diff
@@ -1,32 +1,20 @@
 import * as classnames from "classnames";
-import * as PropTypes from "prop-types";
 import * as React from "react";
 import * as ReactDOM from "react-dom";
 
 import "./Radio.less";
 
-export interface Props {
+export interface Props<IDType extends string = string, ValueType = any> {
   checked?: boolean;
   children: React.ReactNode;
   className?: string;
   disabled?: boolean;
-  id: string;
-  onSelect?: (id: string, value: any) => void;
+  id: IDType;
+  onSelect?: (id: IDType, value: ValueType) => void;
   tabIndex?: number;
-  value?: any;
+  value?: ValueType;
 }
 
-const propTypes = {
```

Comment:
For reasons I don't totally understand, having prop types was causing type errors. I think in these cases it's fine to just remove the propTypes since we don't really get any extra value out of them.